appendix-basic65.tex

```diff
@@ -5579,15 +5669,15 @@ \subsection{LEFT\$}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$C8
 \item [Format:] {\bf LEFT\$(}string{\bf,} n{\bf)}
-\item [Usage:] Returns a string
-               containing the first {\bf n} characters from the
+\item [Usage:] The first {\bf n} characters from the
                argument {\bf string}.
+
                If the length of {\bf string} is equal to or less than {\bf n},
                the resulting string will be identical to the argument string.
 
-               {\bf string} a string expression.
+               {\bf string} a string expression
 
-               {\bf n} a numeric expression (0-255).
+               {\bf n} a numeric expression (0-255)
```

Comment:
Saw you had taken the "(0 -- 255)" double-hyphen approach in other locations, so thought maybe this ought to change that way too.

Reply:
Thanks for mentioning these! I did a regex search and I believe I caught them all (avoiding the ones that are actually in code samples, of course).